app-backend/db/src/main/scala/com/azavea/rf/database/Dao.scala

```diff
@@ -146,6 +146,10 @@ object Dao {
         .getOrElse(throw new Exception("Unsafe delete - delete requires filters"))
         .run
     }
+
+    def exists(id: UUID): ConnectionIO[Boolean] = {
+      (fr"SELECT 1 FROM" ++ tableF ++ fr"WHERE id = ${id}").query[Int].list.map(!_.isEmpty)
```

Comment:
`query.filter(id).selectOption` does the same thing without writing new sql strings, right? like, are we concerned about returning a row vs. an integer, or is that probably an acceptable tradeoff to avoid writing new sql strings?

Reply:
I think doing a `list` here with a limit of 1 may be better for performance. Thoughts?